### ISSUE TITLE
Ask for original images of training set and test set.

### ISSUE THREAD

**Issue by ZhangXu0963** [XLearning-SCU/2021-NeurIPS-NCR#5]
我希望做一个end to end的方法，但很多网址已经失效了，希望作者能分享一下原始图片，非常感谢

**Comment by hi-zhenyu** [comment]
CC数据集包含从网络上获取的图像，Google也仅提供了url下载而不是原始图像，需要研究人员自己去对应网址下载。我不确定我是否有权利进行传播，因此没办法提供相应的原始图片。

如果失效图片不是特别多的话，建议按照论文相同方式随机替换可用的图片，数据集应该不会有特别大的差异。
